Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 29/08/2018 18:05:19,196 ERROR PontoException:35 - Problema no arquivo ponto.properties.
java.lang.ArrayIndexOutOfBoundsException: 1
	at br.gov.ce.gabgov.batida.content.CadastroFrame.carregarArquivoPropriedades(CadastroFrame.java:76)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.<init>(CadastroFrame.java:46)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.getInstancia(CadastroFrame.java:24)
	at br.gov.ce.gabgov.batida.content.PrincipalCadastro.main(PrincipalCadastro.java:21)
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 29/08/2018 18:06:23,098 ERROR JDBCExceptionReporter:78 - Conexão negada. Verifique se o nome da máquina e a porta estão corretos e se o postmaster está aceitando conexões TCP/IP.
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 29/08/2018 18:06:44,160 ERROR PontoException:35 - Sistema apresentou inconsistência. Contate o administrador do sistema.
org.hibernate.exception.JDBCConnectionException: Cannot open connection
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:74)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:43)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:29)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:426)
	at org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:144)
	at org.hibernate.jdbc.BorrowedConnectionProxy.invoke(BorrowedConnectionProxy.java:50)
	at com.sun.proxy.$Proxy12.prepareStatement(Unknown Source)
	at br.gov.ce.gabgov.batida.dao.LotacaoDAO.find(LotacaoDAO.java:44)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.getLotacoes(FuncionarioCadastrarDialog.java:700)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.setarListas(FuncionarioCadastrarDialog.java:748)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.<init>(FuncionarioCadastrarDialog.java:66)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.<init>(CadastroFrame.java:58)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.getInstancia(CadastroFrame.java:24)
	at br.gov.ce.gabgov.batida.content.PrincipalCadastro.main(PrincipalCadastro.java:21)
Caused by: org.postgresql.util.PSQLException: Conexão negada. Verifique se o nome da máquina e a porta estão corretos e se o postmaster está aceitando conexões TCP/IP.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:122)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:65)
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:116)
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:30)
	at org.postgresql.jdbc3.Jdbc3Connection.<init>(Jdbc3Connection.java:24)
	at org.postgresql.Driver.makeConnection(Driver.java:369)
	at org.postgresql.Driver.connect(Driver.java:245)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.hibernate.connection.DriverManagerConnectionProvider.getConnection(DriverManagerConnectionProvider.java:110)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:423)
	... 10 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at org.postgresql.core.PGStream.<init>(PGStream.java:60)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:77)
	... 20 more
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:35:18,114 ERROR PontoException:35 - Problema no arquivo ponto.properties.
java.lang.ArrayIndexOutOfBoundsException: 1
	at br.gov.ce.gabgov.batida.content.CadastroFrame.carregarArquivoPropriedades(CadastroFrame.java:76)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.<init>(CadastroFrame.java:46)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.getInstancia(CadastroFrame.java:24)
	at br.gov.ce.gabgov.batida.content.PrincipalCadastro.main(PrincipalCadastro.java:21)
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:35:22,621 ERROR PontoException:35 - Problema no arquivo ponto.properties.
java.lang.ArrayIndexOutOfBoundsException: 1
	at br.gov.ce.gabgov.batida.content.CadastroFrame.carregarArquivoPropriedades(CadastroFrame.java:76)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.<init>(CadastroFrame.java:46)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.getInstancia(CadastroFrame.java:24)
	at br.gov.ce.gabgov.batida.content.PrincipalCadastro.main(PrincipalCadastro.java:21)
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:35:29,734 ERROR PontoException:35 - Problema no arquivo ponto.properties.
java.lang.ArrayIndexOutOfBoundsException: 1
	at br.gov.ce.gabgov.batida.content.CadastroFrame.carregarArquivoPropriedades(CadastroFrame.java:76)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.<init>(CadastroFrame.java:46)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.getInstancia(CadastroFrame.java:24)
	at br.gov.ce.gabgov.batida.content.PrincipalCadastro.main(PrincipalCadastro.java:21)
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:36:22,383 ERROR JDBCExceptionReporter:78 - Conexão negada. Verifique se o nome da máquina e a porta estão corretos e se o postmaster está aceitando conexões TCP/IP.
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:36:26,096 ERROR JDBCExceptionReporter:78 - Conexão negada. Verifique se o nome da máquina e a porta estão corretos e se o postmaster está aceitando conexões TCP/IP.
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:36:33,229 ERROR JDBCExceptionReporter:78 - Conexão negada. Verifique se o nome da máquina e a porta estão corretos e se o postmaster está aceitando conexões TCP/IP.
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:36:43,395 ERROR PontoException:35 - Sistema apresentou inconsistência. Contate o administrador do sistema.
org.hibernate.exception.JDBCConnectionException: Cannot open connection
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:74)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:43)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:29)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:426)
	at org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:144)
	at org.hibernate.jdbc.BorrowedConnectionProxy.invoke(BorrowedConnectionProxy.java:50)
	at com.sun.proxy.$Proxy12.prepareStatement(Unknown Source)
	at br.gov.ce.gabgov.batida.dao.LotacaoDAO.find(LotacaoDAO.java:44)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.getLotacoes(FuncionarioCadastrarDialog.java:700)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.setarListas(FuncionarioCadastrarDialog.java:748)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.<init>(FuncionarioCadastrarDialog.java:66)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.<init>(CadastroFrame.java:58)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.getInstancia(CadastroFrame.java:24)
	at br.gov.ce.gabgov.batida.content.PrincipalCadastro.main(PrincipalCadastro.java:21)
Caused by: org.postgresql.util.PSQLException: Conexão negada. Verifique se o nome da máquina e a porta estão corretos e se o postmaster está aceitando conexões TCP/IP.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:122)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:65)
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:116)
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:30)
	at org.postgresql.jdbc3.Jdbc3Connection.<init>(Jdbc3Connection.java:24)
	at org.postgresql.Driver.makeConnection(Driver.java:369)
	at org.postgresql.Driver.connect(Driver.java:245)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.hibernate.connection.DriverManagerConnectionProvider.getConnection(DriverManagerConnectionProvider.java:110)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:423)
	... 10 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at org.postgresql.core.PGStream.<init>(PGStream.java:60)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:77)
	... 20 more
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:36:47,109 ERROR PontoException:35 - Sistema apresentou inconsistência. Contate o administrador do sistema.
org.hibernate.exception.JDBCConnectionException: Cannot open connection
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:74)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:43)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:29)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:426)
	at org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:144)
	at org.hibernate.jdbc.BorrowedConnectionProxy.invoke(BorrowedConnectionProxy.java:50)
	at com.sun.proxy.$Proxy12.prepareStatement(Unknown Source)
	at br.gov.ce.gabgov.batida.dao.LotacaoDAO.find(LotacaoDAO.java:44)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.getLotacoes(FuncionarioCadastrarDialog.java:700)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.setarListas(FuncionarioCadastrarDialog.java:748)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.<init>(FuncionarioCadastrarDialog.java:66)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.<init>(CadastroFrame.java:58)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.getInstancia(CadastroFrame.java:24)
	at br.gov.ce.gabgov.batida.content.PrincipalCadastro.main(PrincipalCadastro.java:21)
Caused by: org.postgresql.util.PSQLException: Conexão negada. Verifique se o nome da máquina e a porta estão corretos e se o postmaster está aceitando conexões TCP/IP.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:122)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:65)
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:116)
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:30)
	at org.postgresql.jdbc3.Jdbc3Connection.<init>(Jdbc3Connection.java:24)
	at org.postgresql.Driver.makeConnection(Driver.java:369)
	at org.postgresql.Driver.connect(Driver.java:245)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.hibernate.connection.DriverManagerConnectionProvider.getConnection(DriverManagerConnectionProvider.java:110)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:423)
	... 10 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at org.postgresql.core.PGStream.<init>(PGStream.java:60)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:77)
	... 20 more
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:36:54,243 ERROR PontoException:35 - Sistema apresentou inconsistência. Contate o administrador do sistema.
org.hibernate.exception.JDBCConnectionException: Cannot open connection
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:74)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:43)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:29)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:426)
	at org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:144)
	at org.hibernate.jdbc.BorrowedConnectionProxy.invoke(BorrowedConnectionProxy.java:50)
	at com.sun.proxy.$Proxy12.prepareStatement(Unknown Source)
	at br.gov.ce.gabgov.batida.dao.LotacaoDAO.find(LotacaoDAO.java:44)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.getLotacoes(FuncionarioCadastrarDialog.java:700)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.setarListas(FuncionarioCadastrarDialog.java:748)
	at br.gov.ce.gabgov.batida.content.FuncionarioCadastrarDialog.<init>(FuncionarioCadastrarDialog.java:66)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.<init>(CadastroFrame.java:58)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.getInstancia(CadastroFrame.java:24)
	at br.gov.ce.gabgov.batida.content.PrincipalCadastro.main(PrincipalCadastro.java:21)
Caused by: org.postgresql.util.PSQLException: Conexão negada. Verifique se o nome da máquina e a porta estão corretos e se o postmaster está aceitando conexões TCP/IP.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:122)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:65)
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:116)
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:30)
	at org.postgresql.jdbc3.Jdbc3Connection.<init>(Jdbc3Connection.java:24)
	at org.postgresql.Driver.makeConnection(Driver.java:369)
	at org.postgresql.Driver.connect(Driver.java:245)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.hibernate.connection.DriverManagerConnectionProvider.getConnection(DriverManagerConnectionProvider.java:110)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:423)
	... 10 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at org.postgresql.core.PGStream.<init>(PGStream.java:60)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:77)
	... 20 more
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:37:43,367 ERROR JDBCExceptionReporter:78 - Conexão negada. Verifique se o nome da máquina e a porta estão corretos e se o postmaster está aceitando conexões TCP/IP.
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 12/09/2018 09:38:04,394 ERROR PontoException:35 - Sistema apresentou inconsistência. Contate o administrador do sistema.
org.hibernate.exception.JDBCConnectionException: Cannot open connection
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:74)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:43)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:29)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:426)
	at org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:144)
	at org.hibernate.jdbc.AbstractBatcher.prepareQueryStatement(AbstractBatcher.java:139)
	at org.hibernate.loader.Loader.prepareQueryStatement(Loader.java:1547)
	at org.hibernate.loader.Loader.doQuery(Loader.java:673)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:236)
	at org.hibernate.loader.Loader.doList(Loader.java:2220)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2104)
	at org.hibernate.loader.Loader.list(Loader.java:2099)
	at org.hibernate.loader.criteria.CriteriaLoader.list(CriteriaLoader.java:94)
	at org.hibernate.impl.SessionImpl.list(SessionImpl.java:1569)
	at org.hibernate.impl.CriteriaImpl.list(CriteriaImpl.java:283)
	at br.gov.ce.gabgov.batida.dao.FuncionarioDAO.find(FuncionarioDAO.java:50)
	at br.gov.ce.gabgov.batida.content.FuncionarioConsultarDialog.pesquisar(FuncionarioConsultarDialog.java:243)
	at br.gov.ce.gabgov.batida.content.FuncionarioConsultarDialog$1.actionPerformed(FuncionarioConsultarDialog.java:107)
	at javax.swing.AbstractButton.fireActionPerformed(Unknown Source)
	at javax.swing.AbstractButton$Handler.actionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.fireActionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.setPressed(Unknown Source)
	at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
Caused by: org.postgresql.util.PSQLException: Conexão negada. Verifique se o nome da máquina e a porta estão corretos e se o postmaster está aceitando conexões TCP/IP.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:122)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:65)
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:116)
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:30)
	at org.postgresql.jdbc3.Jdbc3Connection.<init>(Jdbc3Connection.java:24)
	at org.postgresql.Driver.makeConnection(Driver.java:369)
	at org.postgresql.Driver.connect(Driver.java:245)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.hibernate.connection.DriverManagerConnectionProvider.getConnection(DriverManagerConnectionProvider.java:110)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:423)
	... 50 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at org.postgresql.core.PGStream.<init>(PGStream.java:60)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:77)
	... 60 more
Dominio=Casa Civil Num. relogio=1 Hostname=DESKTOP-FS43FV7 :: [BATIDA] 04/10/2018 10:21:00,894 ERROR PontoException:35 - Problema no arquivo ponto.properties.
java.lang.ArrayIndexOutOfBoundsException: 1
	at br.gov.ce.gabgov.batida.content.CadastroFrame.carregarArquivoPropriedades(CadastroFrame.java:76)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.<init>(CadastroFrame.java:46)
	at br.gov.ce.gabgov.batida.content.CadastroFrame.getInstancia(CadastroFrame.java:24)
	at br.gov.ce.gabgov.batida.content.PrincipalCadastro.main(PrincipalCadastro.java:21)
